app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def new
+    @user = current_user
+    @post_id = params[:post_id]
+    @comment = Comment.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: @post, user: @user } }
+    end
+  end
+
+  def create
+    comment_params = params.require(:comment).permit(:text, :post_id)
```

Comment:
- [OPTIONAL] You did great using ``strong parameters`` here, but I'll suggest you extract the `comment_params` into its own private function, this will make your code more modular and readable. here's a snippet of what it should look like. :+1: 

```ruby
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```